NEWS.md

```diff
@@ -732,6 +732,9 @@ Deprecated or removed
   * `trues(A::AbstractArray)` and `falses(A::AbstractArray)` are deprecated in favor of
     `trues(size(A))` and `falses(size(A))` respectively ([#24595]).
 
+  * `workspace` is discontinued, check out [Revise.jl](https://github.com/timholy/Revise.jl)
+    for an alternative workflow ([#25046]).
```

Comment:
😱  Revise.jl still doesn't work on FreeBSD

Reply:
issue filed.
https://github.com/timholy/Revise.jl/issues/66

Reply:
@StefanKarpinski shouldn't this needed a deprecation warning too, not just deleting it?

Else people will just get an `UndefVarError`, with no idea that they should be checking out Revise.